GTMAppAuth/Sources/GTMKeychain.m

```diff
@@ -20,15 +20,14 @@
 
 #import <Security/Security.h>
 
-typedef NS_ENUM(NSInteger, GTMAppAuthFetcherAuthorizationGTMAppAuthGTMOAuth2KeychainError) {
-  GTMAppAuthGTMOAuth2KeychainErrorBadArguments = -1301,
-  GTMAppAuthGTMOAuth2KeychainErrorNoPassword = -1302
-};
-
 /*! @brief Keychain helper class.
  */
 @interface GTMAppAuthGTMOAuth2Keychain : NSObject
 
+// When set to YES, all Keychain queries will have
+// kSecUseDataProtectionKeychain set to true on macOS 10.15+.  Defaults to NO.
+@property(nonatomic) BOOL dataProtectionKeychain;
```

Comment:
This name still doesn't convey to me the purpose of the property. `dataProtectionKeychain` sounds to me like an instance of some type. This property's value is passed to `kSecUseDataProtectionKeychain`, right? I'd suggest `useDataProtectionKeychain` to match what is in the keychain.

Reply:
I like it.  Renamed.